mlir/utils/jenkins/Jenkinsfile

```diff
@@ -790,6 +808,20 @@ pipeline {
                             }
                         }
                     }
+                    stage("Build Composable Kernels") {
```

Comment:
If you look above, this runs when a nighlty param is set.
So in Jenkins, you would need to re-run the job with nighlty enabled to test this..

You can make sure looking at the logs to see whether the stage has executed.